### PR TITLE
Finalize JuMP README

### PR DESCRIPTION
**Pull request by pulsipher**
This addresses the concerns of @odow raised in #311. 

@odow: Any other concerns or suggestions?

### CODE REVIEW

**Comment by odow** [comment]
Let's wait for the preview. It's easy to miss things just looking at Markdown.

**Comment by odow** [comment]
The linter failed. Here are the errors:
```
  {"message": "[Google.Latin] Use 'for example' instead of 'e.g.,'.", "location": {"path": "docs/src/packages/InfiniteOpt.md", "range": {"start": {"line": 17, "column": 24}}}, "severity": "ERROR"}
  {"message": "[Google.Latin] Use 'for example' instead of 'e.g.,'.", "location": {"path": "docs/src/packages/InfiniteOpt.md", "range": {"start": {"line": 19, "column": 44}}}, "severity": "ERROR"}
  {"message": "[Google.Latin] Use 'for example' instead of 'e.g.,'.", "location": {"path": "docs/src/packages/InfiniteOpt.md", "range": {"start": {"line": 20, "column": 16}}}, "severity": "ERROR"}
  {"message": "[Google.Latin] Use 'for example' instead of 'e.g.,'.", "location": {"path": "docs/src/packages/InfiniteOpt.md", "range": {"start": {"line": 21, "column": 13}}}, "severity": "ERROR"}
  {"message": "[Google.Ordinal] Spell out all ordinal numbers ('1st') in text.", "location": {"path": "docs/src/packages/InfiniteOpt.md", "range": {"start": {"line": 22, "column": 3}}}, "severity": "ERROR"}
  {"message": "[Google.Units] Put a nonbreaking space between the number and the unit in '1s'.", "location": {"path": "docs/src/packages/InfiniteOpt.md", "range": {"start": {"line": 22, "column": 3}}}, "severity": "ERROR"}
  {"message": "[Google.Exclamation] Don't use exclamation points in text.", "location": {"path": "docs/src/packages/InfiniteOpt.md", "range": {"start": {"line": 27, "column": 32}}}, "severity": "ERROR"}
  {"message": "[Google.Exclamation] Don't use exclamation points in text.", "location": {"path": "docs/src/packages/InfiniteOpt.md", "range": {"start": {"line": 47, "column": 35}}}, "severity": "ERROR"}
```

**Comment by odow** [comment]
Here's the preview: https://jump.dev/JuMP.jl/previews/PR3343/packages/InfiniteOpt/

**Comment by pulsipher** [comment]
Ok, the latest commit should fix the linter problems

**Comment by odow** [comment]
Started a new doc build off `jump-readme`.

**Comment by odow** [comment]
I think we're safe to skip a doc rebuild with that last commit :smile:

File: docs/jump/README.md [hunk 1]
```diff
@@ -34,17 +34,15 @@ to synthesize new optimization paradigms!
...
 
 ```julia
-julia> ]
-
-(v1.8) pkg> add InfiniteOpt
+julia> import Pkg; Pkg.add("InfiniteOpt")
 ```
 
 ## Documentation
 [![](https://img.shields.io/badge/docs-stable-blue.svg)](https://infiniteopt.github.io/InfiniteOpt.jl/stable)
```
Comment (odow) [inline docs/jump/README.md:41]: I wonder if you want to remove some of these badges. They don't add anything that isn't covered by the link below.

**Comment by odow** [comment]
You could also change the opening sentence to
```
[InfiniteOpt.jl](https://github.com/infiniteopt/InfiniteOpt.jl) is a `JuMP` extension
```

File: docs/jump/README.md (hunk 1 above)
Comment (pulsipher) [reply docs/jump/README.md]: Done

**Comment by pulsipher** [comment]
@odow: anything else before I merge this?

**Review by odow — APPROVED** [review]

**Pull request merged**